### ISSUE TITLE
Support period in record identifiers

### ISSUE THREAD

**Issue by minimal** [dhall-lang/dhall-lang#76]
Like #64 but for `.`. I am considering using dhall to generate kafka connect json but most of the keys have periods in e.g. `"table.name.format"`, `"connection.url"`. Is it reasonable to whitelist period as well?

Thanks

**Comment by Gabriel439** [comment]
Yeah, that seems reasonable to me.  Do you feel comfortable creating a pull request to update the official grammar?  If not, I can do so

**Comment by minimal** [comment]
Sure, that looks simple enough, will do.
